Perfection.java

```diff
@@ -0,0 +1,42 @@
+package main.java.payapal;
+
+public class Perfection {
+
+    private static Perfection perf;
+
+    public synchronized static Perfection getPerf() {
```

Comment:
If I understood correctly, this class is creating a singleton. So It probably would be better to change the name of this method to something like "getInstance" to keep it clearer. https://www.baeldung.com/java-singleton